libs/java.json

```diff
@@ -236,5 +236,14 @@
   {"id": "java.packr", "imports": ["com.badlogicgames.packr."], "name": "packr", "repo": "libgdx/packr", "tags": ["executable-jar"], "tech": ["software-distribution"]},
   {"id": "java.nudge4j", "imports": ["nudge4j."], "name": "nudge4j", "repo": "lorenzoongithub/nudge4j", "tags": ["jvm-control"], "tech": ["diagnostics"]},
   {"id": "java.sliding-menu", "imports": ["com.jeremyfeinstein.slidingmenu."], "name": "SlidingMenu", "repo": "jfeinstein10/SlidingMenu", "tags": ["menus"], "tech": ["android-ui"]},
-  {"id": "java.resilience4j", "imports": ["io.github.resilience4j."], "name": "Resilience4j", "repo": "resilience4j/resilience4j", "tags": ["fault-tolerance"], "tech": ["reliable-code"]}
+  {"id": "java.resilience4j", "imports": ["io.github.resilience4j."], "name": "Resilience4j", "repo": "resilience4j/resilience4j", "tags": ["fault-tolerance"], "tech": ["reliable-code"]},
+  {"id": "java.materialdrawer", "imports": ["com.mikepenz.materialdrawer."], "name": "MaterialDrawer", "repo": "mikepenz/MaterialDrawer", "tags": ["menus"], "tech": ["android-ui"]},
+  {"id": "java.calligraphy", "imports": ["uk.co.chrisjenx.calligraphy."], "name": "Calligraphy", "repo": "chrisjenx/Calligraphy", "tags": ["utils"], "tech": ["android-ui"]},
+  {"id": "java.timber", "imports": ["timber.log."], "name": "Timber", "repo": "JakeWharton/timber", "tags": ["utils"], "tech": ["android"]},
+  {"id": "java.android-image-cropper", "imports": ["com.theartofdev.edmodo.cropper."], "name": "Android Image Cropper", "repo": "ArthurHub/Android-Image-Cropper", "tags": ["utils"], "tech": ["android-ui"]},
+  {"id": "java.easypermissions", "imports": ["pub.devrel.easypermissions."], "name": "EasyPermissions", "repo": "googlesamples/easypermissions", "tags": ["utils"], "tech": ["android"]},
+  {"id": "java.storio", "imports": ["com.pushtorefresh.storio3."], "name": "StorIO", "repo": "pushtorefresh/storio", "tags": ["database","reactive"], "tech": ["android"]},
```

Comment:
tech == storage, tags
tags == android-sql

Not sure what to do with reactive. For now, we only show the first tag. We can keep it there for future use.